test/bin/common.sh

```diff
@@ -87,6 +87,17 @@ SCENARIO_INFO_DIR="${SCENARIO_INFO_DIR:-${IMAGEDIR}/scenario-info}"
 # The CI system will override this depending on the job its running.
 SCENARIO_SOURCES="${SCENARIO_SOURCES:-${TESTDIR}/scenarios}"
 
+# Directory where all the scenarios will be copied for execution
+#
+# shellcheck disable=SC2034  # used elsewhere
+SCENARIOS_TO_RUN="${OUTPUTDIR}/scenarios-to-run"
```

Comment:
Most everything else we generate goes under IMAGEDIR so it can be collected as part of the artifacts for the job. Should we include these files, too?

Reply:
This is visible in the logs, only in a different form:
```bash
+ 15:26:34.790026416 microshift/test/bin/ci_phase_iso_boot.sh:86 	cat /home/ec2-user/microshift/_output/test-images/launch_vm_jobs.txt
Seq	Host	Starttime	JobRuntime	Send	Receive	Exitval	Signal	Command
1	:	1709911319.000	   135.000	0	216600	0	0	bash -x ./bin/scenario.sh create '/home/ec2-user/microshift/_output/scenarios-to-run/el92-base@upgrade-ok.sh'
2	:	1709911324.000	   137.000	0	216229	0	0	bash -x ./bin/scenario.sh create '/home/ec2-user/microshift/_output/scenarios-to-run/el92-crel@upgrade-ok.sh'
3	:	1709911329.000	   134.000	0	218019	0	0	bash -x ./bin/scenario.sh create '/home/ec2-user/microshift/_output/scenarios-to-run/el92-prel@upgrade-ok.sh'
4	:	1709911334.000	   135.000	0	225317	0	0	bash -x ./bin/scenario.sh create '/home/ec2-user/microshift/_output/scenarios-to-run/el92-src@backup-and-restore-on-reboot.sh'
5	:	1709911339.000	   131.000	0	216191	0	0	bash -x ./bin/scenario.sh create '/home/ec2-user/microshift/_output/scenarios-to-run/el92-src@backups.sh'
6	:	1709911344.000	   135.000	0	218452	0	0	bash -x ./bin/scenario.sh create '/home/ec2-user/microshift/_output/scenarios-to-run/el92-src@downgrade-block.sh'
7	:	1709911349.000	   135.000	0	223335	0	0	bash -x ./bin/scenario.sh create '/home/ec2-user/microshift/_output/scenarios-to-run/el92-src@log-scan.sh'
8	:	1709911354.000	   144.000	0	220439	0	0	bash -x ./bin/scenario.sh create '/home/ec2-user/microshift/_output/scenarios-to-run/el92-src@multi-nic.sh'
10	:	1709911364.000	   135.000	0	217983	0	0	bash -x ./bin/scenario.sh create '/home/ec2-user/microshift/_output/scenarios-to-run/el92-src@standard-suite1.sh'
11	:	1709911369.000	   135.000	0	219334	0	0	bash -x ./bin/scenario.sh create '/home/ec2-user/microshift/_output/scenarios-to-run/el92-src@standard-suite2.sh'
12	:	1709911374.000	   133.000	0	218789	0	0	bash -x ./bin/scenario.sh create '/home/ec2-user/microshift/_output/scenarios-to-run/el92-src@storage.sh'
13	:	1709911379.000	   138.000	0	226159	0	0	bash -x ./bin/scenario.sh create '/home/ec2-user/microshift/_output/scenarios-to-run/el92-src@upgrade-fails-on-1st-boot-but-recovers-on-2nd.sh'
14	:	1709911384.000	   134.000	0	220392	0	0	bash -x ./bin/scenario.sh create '/home/ec2-user/microshift/_output/scenarios-to-run/el92-src@upgrade-fails.sh'
15	:	1709911389.000	   137.000	0	222884	0	0	bash -x ./bin/scenario.sh create '/home/ec2-user/microshift/_output/scenarios-to-run/el92-yminus2@upgrade-ok.sh'
9	:	1709911359.000	   235.000	0	257790	0	0	bash -x ./bin/scenario.sh create '/home/ec2-user/microshift/_output/scenarios-to-run/el92-src@rpm-install.sh'
```

Reply:
That gives us the list of scenarios that ran. I guess we don't expect those files to change, so there's not much benefit to having their content captured.